app/src/main/java/org/mifos/mobilebanking/ui/fragments/SavingAccountsTransactionFragment.java

```diff
@@ -338,7 +337,7 @@ private void filter(long startDate , long endDate) {
 
         dummyTransactionList = new ArrayList<>(transactionsList);
         savingAccountsTransactionPresenter.filterTransactionList(dummyTransactionList ,
-                                                                    startDate , endDate);
+                startDate , endDate);
```

Comment:
Undo this change

Reply:
Android Studio is doing that, not me. I will fix that but I suggest to let the next things like those ones be merged. If Android Studio is doing that, it's because of some reason...